tools/syncRepository.sh

```diff
@@ -67,7 +67,9 @@ then
     pullAndSync $target common
     pullAndSync $target webProfile6
```

Comment:
Should webProfile6 removed from sync? Not entirely sure.

Reply:
I think so.